src/wallet.rs

```diff
@@ -208,6 +208,60 @@ where
 		Ok(self.inner.lock().unwrap().get_balance()?)
 	}
 
+	pub(crate) fn send_to_address(
```

Comment:
Perhaps it is better to allow pass an optional fee rate?
Also a lightning node user would probably want to send all the onchain funds (empty the wallet) so perhaps this use case should be taken into account when exposing this ability at the Node level.